webroot/js/local.js

```diff
@@ -19,4 +19,40 @@ jQuery(document).on('ready', function() {
       date: new Date($(this).data('timestamp') * 1000)
     });
   });
+
+  $('input.autocomplete, select.autocomplete').each(function (i, e) {
```

Comment:
Should we make use of role? I mean we're using Bootstrap for this plugin and they use `role` quite constantly to identify UI components. Another thing is that it would still work even if someone changes the styles.

On that note, we should think of standardizing the selectors.

/cc @lorenzo @ADmad @josegonzalez @jippi